extrahop_consts.py

```diff
@@ -13,37 +13,105 @@
 # either express or implied. See the License for the specific language governing permissions
 # and limitations under the License.
 
+# Tokens and state file
+EXTRAHOP_OAUTH_TOKEN_STRING = "token"
+EXTRAHOP_OAUTH_ACCESS_TOKEN_STRING = "access_token"
+EXTRAHOP_OAUTH_ACCESS_TOKEN_IS_ENCRYPTED = "is_encrypted"
+EXTRAHOP_DECRYPTION_ERROR = "Error occurred while decrypting the state file"
+EXTRAHOP_ENCRYPTION_ERROR = "Error occurred while encrypting the state file"
+EXTRAHOP_DETECTION_LAST_INGESTED_TIME = "detection_last_ingested_time"
+EXTRAHOP_METRICS_LAST_INGESTED_TIME = "metrics_last_ingested_time"
+EXTRAHOP_PACKETS_LAST_INGESTED_TIME = "packets_last_ingested_time"
+
 # Error and status messages
-EXTRAHOP_ERR_MSG_UNAVAILABLE = 'Error message unavailable. Please check the asset configuration and|or action parameters'
+EXTRAHOP_ERROR_MESSAGE_UNAVAILABLE = 'Error message unavailable. Please check the asset configuration and|or action parameters'
 EXTRAHOP_DEBUG_REST_ENDPOINT = "Making REST call to {}"
-EXTRAHOP_STATE_FILE_CORRUPT_ERR = "Error occurred while loading the state file due to its unexpected format. " \
+EXTRAHOP_STATE_FILE_CORRUPT_ERROR = "Error occurred while loading the state file due to its unexpected format. " \
     "Resetting the state file with the default format."
+EXTRAHOP_CONTAINER_LIMIT_ERROR = "Container count should be greater than or equal to the number of object_ids"
+EXTRAHOP_METRIC_REQUIRED_PARAM = "Required parameters: metric_cycle_length, object_type, object_id, metric_category and metric_name"
+EXTRAHOP_METRIC_REQUIRED_API_PARAM = "API required parameters: cycle, object_type, object_ids, metric_category and metric_specs"
+EXTRAHOP_METRIC_NOT_NEEDED_MESSAGE = "For asset_id: {}, Interval should be in multiple of cycle length"
```

Comment:
```suggestion
EXTRAHOP_METRIC_NOT_NEEDED_MESSAGE = "For asset_id: {}, Interval should be in multiples of cycle length"
```
Kind of confusing wording